examples/benchmarks/bert/src/text_data.py

```diff
@@ -266,8 +261,6 @@ def build_text_dataloader(
                     cfg.dataset.get('validate_hash', None),
                     keep_zip=stream.get('keep_zip', None) or
                     cfg.dataset.get('keep_zip', False),
-                    keep_raw=stream.get('keep_raw', None) or
```

Comment:
Just noting that this is correct and that `keep_raw` is no longer a flag in `mosaicml-streaming` (see Streaming [docs](https://docs.mosaicml.com/projects/streaming/en/stable/api_reference/generated/streaming.Stream.html#stream))

Reply:
can you check that the defaults here match the defaults currently set in llm foundry?

Reply:
The defaults in llm foundry are a bit different. Should we update this function whole-hog?

From llmfoundry [text_data.py](https://github.com/mosaicml/llm-foundry/blob/main/llmfoundry/data/text_data.py)
```python
def __init__(self,
                 tokenizer: PreTrainedTokenizerBase,
                 max_seq_len: int,
                 streams: Optional[Sequence[Stream]] = None,
                 remote: Optional[str] = None,
                 local: Optional[str] = None,
                 split: Optional[str] = None,
                 download_retry: int = 2,
                 download_timeout: float = 60,
                 validate_hash: Optional[str] = None,
                 keep_zip: bool = False,
                 epoch_size: Optional[Union[int, str]] = None,
                 predownload: Optional[int] = None,
                 cache_limit: Optional[Union[int, str]] = None,
                 partition_algo: str = 'relaxed',
                 num_canonical_nodes: Optional[int] = None,
                 batch_size: Optional[int] = None,
                 shuffle: bool = False,
                 shuffle_algo: str = 'py1e',
                 shuffle_seed: int = 9176,
                 shuffle_block_size: Optional[int] = None,
                 sampling_method: str = 'balanced',
                 sampling_granularity: int = 1,
                 batching_method: str = 'random',
                 **kwargs: Any):
  ```

Reply:
Since it's still text data, this should be good!